engines/bastion/app/views/bastion/layouts/assets.html.erb

```diff
@@ -19,6 +19,7 @@
     angular.module('Bastion').value('currentLocale', '<%= I18n.locale %>');
     angular.module('Bastion').value('CurrentOrganization', "<%= Organization.current.id if Organization.current %>");
     angular.module('Bastion').value('simpleContentAccessEnabled', <%= Organization.current.simple_content_access? if Organization.current %>);
+    angular.module('Bastion').value('isManifestImported', <%= !!Organization.current&.owner_details&.[](:upstreamConsumer)&.[](:webUrl) %>)
```

Comment:
Can you move the logic to a method in owner.rb
```suggestion
    angular.module('Bastion').value('isManifestImported', <%= Organization.current&.manifest_imported? %>)
```

Reply:
the method itself already exists! https://github.com/Katello/katello/blob/master/app/models/katello/concerns/organization_extensions.rb#L94

Reply:
@parthaa updated!

Reply:
@jlsherrill good catch!  Deleted the method I wrote in glue, which used
```rb
owner_details&.[](:upstreamConsumer)&.[](:webUrl)
```
in favor of the existing method which only uses
```rb
owner_details['upstreamConsumer'].present?
```

Now we're using two different sets of logic to decide if a manifest is imported. Should I also change the React selector to use the latter logic?

Reply:
cc @jturel I feel like you were the one who originally used `webUrl`..

Reply:
also i noticed in this pr: https://github.com/Katello/katello/pull/8859/files#diff-3f8d4b7127ce10e6e2bc95774bd0b8df3822f0c4075e08efd96d4b2fae6f39b4R38-R41  that some caching comes into play, i presume to reduce calls to candlepin.  I wonder if we should do that here since this is happening on every angular page load (some background on that change from @jturel and @chris1984 might be good if they remember)

Reply:
> cc @jturel I feel like you were the one who originally used `webUrl`..

I don't think so because `&.[]` is not in my repertoire but I did write `manifest_imported?`! I'm +1 to using that method here. I also think you could enhance it to use caching similar to the `simple_content_access?` method. However, I'd make the the default `cached: false` so that the whole app doesn't need to be audited to ensure that it's not incorrectly reading a cached value when it really needs an uncached value. So you can just call it with `cached: true` here

Reply:
Updated :+1: